sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -321,6 +330,162 @@ directive:
     where: $.parameters
     transform: >
       $.OpenIdConnectIdParameter['x-ms-client-name'] = 'OpenId';
+      $.IfMatchOptionalParameter['x-ms-format'] = 'etag';
+      $.IfMatchRequiredParameter['x-ms-format'] = 'etag';
+  - from: apimgroups.json
```

Comment:
These directives are confusing, could you please add some comments here to describe what are these things for?

Reply:
Added.